### PR TITLE
When an HTTP error occurs with noStore, return the error

### PR DESCRIPTION
**Pull request by jkdufair**
Ready for review!

### CODE REVIEW

File: src/noStoreSaga.ts [hunk 1]
```diff
@@ -70,6 +70,11 @@ export function* handleAction(action: any): SagaIterator {
 		return
 	}
 
+	if (failedResult) {
+		hook(failedResult)
+		return
+	}
+
 	if (!receivedResult || !receivedResult.data || !!failedResult) {
```
Comment (builtbywill) [inline src/noStoreSaga.ts:78]: @jkdufair should we remove the ` || !!failedResult` here since it isn't need anymore?

Comment (jkdufair) [reply src/noStoreSaga.ts]: Yes we should! Thank you. Updated.

**Review by builtbywill — APPROVED** [review]
@jkdufair seems pretty straight-forward

however, should probably merge this into develop first, and follow the git-flow pattern for a new release

File: src/noStoreSaga.test.ts [hunk 2]
```diff
@@ -233,21 +233,20 @@ describe('handleAction', () => {
...
 
-	test('should call correct hook with `null` on failure', () => {
+	test('should call correct hook with `failedResult` on failure', () => {
 		const action = {
 			modelName: 'someModel',
 			type: NET_ACTION.DATA_REQUESTED,
 			noStore: true,
 			guid: firstKey
 		}
+		const result = { title: "I'm a teapot", code: 418, message: 'Short and stout' }
```
Comment (lily514) [inline src/noStoreSaga.test.ts:243]: :joy: love this test case!

**Review by lily514 — APPROVED** [review]
I agree with @builtbywill that we should follow normal gitflow and merge to develop then master, but this looks straightforward! thanks!!

**Pull request merged**